### PR TITLE
#129/about page

### LINKED ISSUE #129: Update /about page with copy from Gatsby
### Expected Behaviour

When users navigate to the about page on Pathways they should find some information about the purpose of the platform.

### Current Behaviour

It says 'T-Level stuff'

### Any other notes

Gatsby have provided this copy:

Apprenticeships, T Levels and other technical qualifications draw their content from occupational standards set out by employers. Training is structured around occupations so that individuals can develop the knowledge, skills and behaviour they need to do well in those roles. In many cases these occupations are found across multiple sectors in the labour market. For example, software engineers work for banks as part of the finance sector, and there are finance roles such as payroll and accounts assistants working in the health or retail sectors. Thus individual T Levels feed into multiple sectors – there are a range of possibilities for students to progress into, and pathways for employers to recruit from.
 
This tool enables users - including employers and local stakeholders, such as local authorities - to visualise the occupations that make up the sectors they operate in, identify the technical education pathways and specific T Levels that feed into these sectors. Employers can input the occupations they have in their business and see the T Level pathways that link to these roles. This will help employers determine where they can start building their workforce pipelines, for example by beginning to host T Level industry placements in these pathways.

### PR DESCRIPTION
**Pull request by camelPhonso**
# Description

**Closes #129**  

Sets up the '/about' route with the copy provided by Gatsby and '/' as our originally designed `InfoPage.tsx`

### Files changed

- `App.jsx` to set up the new route for `/` 
- `About.jsx`, `About.css` and `young-professionals.jpeg` to display information about Pathways to the users
- `InfoPage.css` refactored to avoid creating clashes with the styling of other pages

### UI changes
`/about`
Screenshot 2024-08-13 at 11 13 00

### Changes to Documentation

n/a

# Tests

Include a description of any tests added - if none have been added explain why.n/a

### CODE REVIEW

File: frontend/src/components/Header.jsx [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
 
 export default function Header({ searchHandler }) {
+  const [currentPage, setCurrentPage] = useState()
+
+  useEffect(() => {
+    const page = window.location.href
+    const pageRegex = /(about)|(search)/g
+
+    setCurrentPage(page.match(pageRegex)[0])
+  }, [window.location.href])
```
Comment (nichgalzin) [inline frontend/src/components/Header.jsx:16]: Getting a eslint error with this dependency error:
React Hook useEffect has an unnecessary dependency: 'window.location.href'. Either exclude it or remove the dependency array. Outer scope values like 'window.location.href' aren't valid dependencies because mutating them doesn't re-render the component.

File: frontend/src/components/Header.jsx [hunk 2]
```diff
@@ -1,24 +1,34 @@
 import { Link } from "react-router-dom"
+import { useEffect, useState } from "react"
 
 import TextSearch from "./TextSearch"
 import gatsbyLogo from "../images/GATSBY.jpeg"
 import "../style/Header.css"
 
 export default function Header({ searchHandler }) {
+  const [currentPage, setCurrentPage] = useState()
```
Comment (nichgalzin) [inline frontend/src/components/Header.jsx:9]: Know this isn't ts. But maybe initialise this state with a value that lets future devs know what to expect. In this case an empty string? Whatcha think?

File: frontend/src/style/Header.css [hunk 3]
```diff
@@ -19,13 +19,8 @@
...
 }
 
-.search-bar {
-  padding: 5px 10px;
-  font-size: 16px;
-  border-radius: 4px;
-  border: 1px solid #ccc;
-  width: 300px;
-  margin: 0 20px;
+.bold {
```
Comment (nichgalzin) [inline frontend/src/style/Header.css:22]: This should go in global.css perhaps as its a universal utility class?

File: frontend/src/pages/About.jsx [hunk 4]
```diff
@@ -1,3 +1,44 @@
+import youngProfessionals from "../images/young-professionals.jpg"
+import "../style/About.css"
+
 export default function About() {
-  return <h1>About Page</h1>
+  return (
+    <>
+      <div className="about-page-container">
```
Comment (nichgalzin) [inline frontend/src/pages/About.jsx:7]: Think this should be a main tag got each page component

File: frontend/src/components/InfoPage.jsx [hunk 5]
```diff
@@ -6,11 +6,11 @@ function InfoPage() {
 
   return (
     <section className="main-section" style={{ backgroundImage: `url(${backgroundImage})` }}>
```
Comment (nichgalzin) [inline frontend/src/components/InfoPage.jsx:8]: Think this should be a main tag got each page component

**Review by nichgalzin — CHANGES_REQUESTED** [review]
Two broader things:
- Can you do mobile styling for the about page?
- The info page seems to be crashing on this branch

File: frontend/src/components/Header.jsx (hunk 1 above)
Comment (camelPhonso) [reply frontend/src/components/Header.jsx:16]: eslint is technically right, except that this useEffect is tracking the url to check which navigation button to highlight. So that does re-render when the location changes, because this component is part of our layout that persists on all pages. 

Removing this dependency breaks that functionality. We could track the params, but those won't apply if we are navigating out of '/search'

File: frontend/src/components/Header.jsx (hunk 2 above)
Comment (camelPhonso) [reply frontend/src/components/Header.jsx:9]: yeah, I'm actually doing it now for the other state as well that just needs to be an array

File: frontend/src/style/Header.css (hunk 3 above)
Comment (camelPhonso) [reply frontend/src/style/Header.css:22]: yes, yes it should

File: frontend/src/pages/About.jsx (hunk 4 above)
Comment (camelPhonso) [reply frontend/src/pages/About.jsx]: done

File: frontend/src/components/InfoPage.jsx (hunk 5 above)
Comment (camelPhonso) [reply frontend/src/components/InfoPage.jsx]: done

**Review by nichgalzin — APPROVED** [review]
Great stuff

**Review by ShaughnAnderson94 — APPROVED** [review]
whatever nich said I agree
